### PR TITLE
DB-9540 Make RegionLoads use bytes instead of MBs

### PR DESCRIPTION
**Pull request by dgomezferro**
Fix RegionSplitsIT and don't require /BAD directory to exist for
SpliceRegionAdminIT

### CODE REVIEW

File: hbase_storage/src/main/java/com/splicemachine/storage/HPartitionLoad.java [hunk 1]
```diff
@@ -19,15 +19,13 @@
  *         Date: 1/7/16
  */
 public class HPartitionLoad implements PartitionLoad{
-    private final int storefileSizeMB;
-    private final int memStoreSizeMB;
-    private final int storefileIndexSizeMB;
+    private final long storefileSize;
+    private final long memStoreSize;
```
Comment (jyuanca) [inline hbase_storage/src/main/java/com/splicemachine/storage/HPartitionLoad.java:23]: Why storefileIndexSize is removed?

File: hbase_storage/src/main/java/com/splicemachine/storage/HServerLoad.java [hunk 2]
```diff
@@ -60,7 +60,7 @@ public long totalRequests(){
         for(Map.Entry<byte[],RegionLoad> regionLoad:regionsLoad.entrySet()){
             String name = Bytes.toString(regionLoad.getKey());
             RegionLoad rl = regionLoad.getValue();
-            PartitionLoad pl = new HPartitionLoad(name,rl.getStorefileSizeMB(),rl.getMemStoreSizeMB(),rl.getStorefileIndexSizeMB());
+            PartitionLoad pl = new HPartitionLoad(name,rl.getStorefileSizeMB()*1024,rl.getMemStoreSizeMB()*1024);
```
Comment (yxia92) [inline hbase_storage/src/main/java/com/splicemachine/storage/HServerLoad.java:63]: Should we multiple 1024*1024 to get the number in bytes?

File: hbase_storage/src/main/java/com/splicemachine/storage/HPartitionLoad.java (hunk 1 above)
Comment (dgomezferro) [reply hbase_storage/src/main/java/com/splicemachine/storage/HPartitionLoad.java:23]: It wasn't used

File: hbase_storage/src/main/java/com/splicemachine/storage/HServerLoad.java (hunk 2 above)
Comment (dgomezferro) [reply hbase_storage/src/main/java/com/splicemachine/storage/HServerLoad.java:63]: Good catch @yxia92 , addressed in a new commit

**Pull request merged**